manifests/platform.yaml

```diff
@@ -128,7 +128,7 @@ spec:
     values:
       imagesHost: dtr.dev.cray.com
   - name: cray-shared-kafka
-    version: 0.4.1
+    version: 0.7.0
```

Comment:
Why the jump from 0.4.1 to 0.7.0?

Reply:
Maybe we should just bump to 0.5.0, here, (for the release/0.9 branch) since the release/1.0 branch has version 0.7.0 and the release/1.2 branch has version 1.0.0?

Reply:
Yes, to avoid another check-in (same content) in 0.9 branch, taken from 1.0 branch version.

Reply:
@djlapenta : 0.5.0(used in 1.0 release earlier)  and 0.6.0(used in 1.2 release earlier) are the version of shared-kafka already been built and available on artifactory so we won't be able to use them again. Please correct me if my understanding wrong here..

@jeremy-duckworth : I learnt that there's no change in shared-kafka between 0.9 and 1.0 so there no point in rebuilding the shared-kafka for 0.9 branch for the same changes what we have done on 1.0. Hence used the same version 0.7.0 in 0.9.x as well. Please let me know if you this doesn't look correct to you.